Demo/Sources/SettingsView.swift

```diff
@@ -87,6 +88,19 @@ struct SettingsView: View {
         }
     }
 
+    @ViewBuilder
+    private func versionSection() -> some View {
```

Comment:
The layout would probably be even nicer if the footer is attached to the last section (currently the debugging one):

```swift
Section {
    // ...
} header: {
    Text("Debugging")
} footer: {
    HStack {
        // ...
    }
    .frame(maxWidth: .infinity)
}
```